Vagrantfile

```diff
@@ -25,6 +25,24 @@ Vagrant.configure("2") do |config|
 # Get the VM up-to-date
     sudo apt-get update
 
+# Install go tools
+    sudo apt-get install git
+    wget https://storage.googleapis.com/golang/go1.8.1.linux-amd64.tar.gz --quiet
```

Comment:
Maybe, but I've gone with the language webpage's recommendation (as we have for OCaml)